WebService/v1/Models/SimulationApiModel/SimulationApiModel.cs

```diff
@@ -274,5 +263,19 @@ public async Task ValidateInputRequestAsync(ILogger log, IIotHubConnectionString
                 await connectionStringManager.ValidateConnectionStringAsync(iotHub.ConnectionString);
             }
         }
+
+        private static SimulationStatistics getSimulationStatistics(Simulation simulation)
```

Comment:
fix method name to start with capital letter

Reply:
done